index.html

```diff
@@ -16,28 +16,33 @@
  
   <div class="nav-bar">
      
-
     <div class="logo"> 
          <a href="#"><img src="/images/My Logo.png" alt="logo for the website"></a> 
-    </div>
-    <ul class="dsk-li">
-      <li> <a href="#">Portfolio</a> </li>
-      <li> <a href="#">about</a></li>
-     <li><a href="#">contact</a></li>
-</ul>
-       
+        </div>
+
+
+         <label for="check" class="hamberger-btn">
+             <img id="ham" href="#" src="./images/Union.png">
+               </label>
+               <ul class="dsk-li">
+                  <li class="nav-btn"> <a href="#">contact</a> </li>
+                   <li class="nav-btn"> <a href="#">about</a></li>
+                   <li class="nav-btn"><a href="#">Portfolio</a></li>
```

Comment:
<img width="1279" alt="1" src="https://github.com/hetrox8/portfolio1/assets/77843050/cf5c8932-4bab-44f7-a454-cc63aa016109">

- [ ] As the previous reviewer said, there is a space surrounding the nav bar that is not present in the figma design, please make sure you remove it.
- [ ] I think while you were bringing these changes, you have broken some thing regarding the image logo as shown above, please fix that